src/test/java/com/github/tomakehurst/wiremock/SavingMappingsAcceptanceTest.java

```diff
@@ -41,8 +44,10 @@ public class SavingMappingsAcceptanceTest extends AcceptanceTestBase {
 
   private static void resetFileSourceRoot() {
     try {
-      if (FILE_SOURCE_ROOT.exists()) {
-        FileUtils.deleteDirectory(FILE_SOURCE_ROOT);
+      if (new File(String.valueOf(FILE_SOURCE_ROOT.toPath().toAbsolutePath())).exists()) {
```

Comment:
I understand the replacement of `FileUtils.deleteDirectory`, but what was wrong with `if (FILE_SOURCE_ROOT.exists())`? It looks like you're taking the `File` instance, turning it into a `Path` and then back into a `File`?

Reply:
done